### ISSUE TITLE
Support ruby 3.1

### ISSUE THREAD

**Issue by yolk** [jgarber623/svgeez#102]
Hi there,

are there any known issues with svgeez working on ruby 3.1? 
The gemspec requires a version below 3.1.

Thanks for this gem!

**Comment by kimili** [comment]
I'm getting the same issue. Since upgrading a project to Ruby 3.1, my automated test process is failing on this:

```
svgeez-4.0.0 requires ruby version < 3.1, >= 2.5, which is incompatible with the
current version, ruby 3.1.0p0
```

Is there any particular reason it's capped at `< 3.1`? Or do we just need to add open up that restriction?

**Comment by jgarber623** [comment]
Thanks for your patience, y'all! Just released v4.1.0:

- [RubyGems](https://rubygems.org/gems/svgeez/versions/4.1.0)
- [GitHub](https://github.com/jgarber623/svgeez/releases/tag/v4.1.0)
